bigdata/src/main/scala/org/w3/banana/bigdata/package.scala

```diff
@@ -0,0 +1,48 @@
+package org.w3.banana
+
+import org.openrdf.model.{URI, Resource, Statement, Value}
+import org.openrdf.query.{GraphQueryResult, BindingSet, TupleQueryResult}
+import org.openrdf.repository.RepositoryResult
+import scala.collection.immutable.{Map, List}
+
+/**
+ * I put here some implicits to add Iterable support for bigdata StatementIterators
+ * as they do not implement Java iterator/iterable so JavaConversions do not work for them
+ */
+package object bigdata extends BigdataResultsConversions
+
+trait BigdataResultsConversions
+{
+
+  /**
+   * Implicit class that turns  query result into iterator (so methods toList, map and so on can be applied to it)
+   * @param results results in sesame format
+   */
+  implicit class TupleResult(results: TupleQueryResult)  extends Iterator[BindingSet]
```

Comment:
Ok. But I will postpone it to later commits as it works well and I have to implement sparql support and some other required features.